client/reader/sidebar/test/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { stub } from 'sinon';
+import mockery from 'mockery';
+
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'ReaderSidebar', ( ) => {
+	let ReaderSidebar
+
+	let shouldRenderAppPromo;
+	const shouldRenderAppPromoDefaultProps = {
+		isDesktopPromoDisabled: false,
+		isUserLocaleEnglish: true,
+		isViewportMobile: false,
+		isUserOnChromeOS: false,
+		isDesktopPromoConfiguredToRun: true,
+		isUserDesktopAppUser: false
+	};
+
+	const readerSidebarDefaultProps = {
+		path: '/',
+	};
+
+	useFakeDom();
+	useMockery();
+
+	before( ( ) => {
+		mockery.registerMock( 'component-closest', stub() );
+		shouldRenderAppPromo = require( '..' ).shouldRenderAppPromo;
+		ReaderSidebar = require( '..' ).ReaderSidebar;
+	} );
```

Comment:
I think that we actually need to include this stuff inside of the call to `useMockery()` and not outside of it in order not to mess with other tests.

maybe @blowery could review this usage, as I am not as confident on how to use it.